openassessment/templates/openassessmentblock/oa_team_uploaded_files.html

```diff
@@ -10,7 +11,9 @@ <h5 class="submission__answer__part__text__title">
         {% endif %}
 
         <div class="submission__answer__files">
-        {% for file_url, file_description, file_name, owner_username in team_file_urls %}
+        {% for file in team_file_urls %}
+        {% with file_url=file|get_item:"download_url" file_description=file|get_item:"description" file_name=file|get_item:"name" owner_username=file|get_item:"uploaded_by" %}
```

Comment:
I don't think we need this?
Does `file.download_url`, `file. description` etc not work?

Reply:
actually .... i think that same exact syntax would work for the namedtuples as well

Reply:
Yup, this works. Will update.